src/GslibIO.jl

```diff
@@ -155,4 +198,51 @@ function save(file::File{format"GSLIB"}, sdata::AbstractData)
        propnames=vars)
 end
 
+# low level function for saving data to a legacy GSLIB format
+function save_legacy(filename::AbstractString, data::AbstractMatrix,
+                     varnames::NTuple, header::AbstractString, na)
+  @assert size(data, 2) == length(varnames) "invalid data for the specified variable names"
+  nvars = size(data, 2)
+
+  # handle missing values
+  datana = replace(data, NaN=>na)
+
+  open(filename, "w") do f
+    write(f, "$header\n")
+    write(f, "$nvars\n")
+    for v in varnames
+      write(f, "$v\n")
+    end
+    writedlm(f, datana, ' ')
+  end
+end
+
+"""
+    save_legacy(filename, sdata)
+
+Save spatial data `sdata` to `filename` in legacy GSLIB format. Optionally, specify
+the `coordnames`, the `header` and the value `na` used to represent missing entries.
+"""
+function save_legacy(filename::AbstractString, sdata::AbstractData; coordnames=(:x, :y, :z),
+                     header="# This file was generated with GslibIO.jl", na=-999.0)
+  table = values(sdata)
+  sdomain = domain(sdata)
+  
+  if sdomain isa PointSet
+    # add coordinates to data and coordnames to varnames
+    coords = coordinates(sdomain)
+    @assert ncoords(sdomain) == length(coordnames) "the length of coordinate names must be equal to the coordinate dimension"
+    varnames = [coordnames...; propertynames(table)]
+    data = [coords' Matrix(table)]
```

Comment:
```suggestion
    @assert ncoords(sdomain) == length(coordnames) "the length of coordinate names must be equal to the coordinate dimension"
    varnames = [coordnames...; propertynames(table)]
    data = [coordinates(sdomain)' Matrix(table)]
```

I thought I had suggested this one already, please fix this intermediate variable here.

Reply:
You did and I'm pretty sure that I did the change. Anyway, it will be fix in the next push.

Reply:
I use to collaborate from two different computers. Maybe I missed to push some commits working in one computer. My bad.